### PR TITLE
chore(bump): bump version of akka for datahub-frontend

### PR DESCRIPTION
**Pull request by david-leifker**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by RyanHolstien — APPROVED** [review]

**Comment by david-leifker** [comment]
Versions are constrained based on akka licensing changes

**Comment by david-leifker** [comment]
Next opportunity will be an upgrade to Play 3.x (https://www.playframework.com/documentation/3.0.x/General#How-Play-Deals-with-Akkas-License-Change) and I believe we are lacking support from the gradle plugin until 3.1 specifically.

File: build.gradle [hunk 1]
```diff
@@ -105,7 +106,14 @@ project.ext.spec = [
...
 
 project.ext.externalDependency = [
-    'akkaHttp': "com.typesafe.akka:akka-http-core_$playScalaVersion:10.2.10",
+    'akkaHttp': "com.typesafe.akka:akka-http-core_$playScalaVersion:10.2.10", // max version due to licensing
+    'akkaActor': "com.typesafe.akka:akka-actor_$playScalaVersion:$akkaVersion",
+    'akkaStream': "com.typesafe.akka:akka-stream_$playScalaVersion:$akkaVersion",
+    'akkaActorTyped': "com.typesafe.akka:akka-actor-typed_$playScalaVersion:$akkaVersion",
+    'akkaSlf4j': "com.typesafe.akka:akka-slf4j_$playScalaVersion:$akkaVersion",
+    'akkaJackson': "com.typesafe.akka:akka-serialization-jackson_$playScalaVersion:$akkaVersion",
+    'akkaParsing': "com.typesafe.akka:akka-parsing_$playScalaVersion:$akkaVersion",
```
Comment (gallyamb) [inline build.gradle:115]: Not all artifacts have a version 2.6.21, e.g. there is no version 2.6.21 for https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-parsing_2.13. Isn't it a problem?

Comment (AlanXAme) [reply build.gradle:115]: I encountered the same problem, the version of akka-parsing_2.13 could be 10.2.10 same as akkaHttp.

**Pull request merged**